### PR TITLE
Add grpc methods for changing sessions and checking if the server is ready

### PR DESCRIPTION
**Pull request by lindkvis**

### CODE REVIEW

File: GrpcInterface/cafGrpcObjectService.cpp [hunk 1]
```diff
@@ -130,9 +130,7 @@ grpc::Status ObjectService::ExecuteMethod( grpc::ServerContext* context, const M
             {
                 if ( method->type() == ObjectMethod::Type::READ_WRITE && session->type() == Session::Type::OBSERVING )
                 {
-                    return grpc::Status( grpc::UNAUTHENTICATED,
-                                         "Observing session '" + request->session().uuid() +
-                                             "' is not valid for non-const methods" );
+                    return grpc::Status( grpc::UNAUTHENTICATED, "Opertion cannot be completed with observing sessions" );
```
Comment (kriben) [inline GrpcInterface/cafGrpcObjectService.cpp:133]: Small type here: Opertion => Operation.

File: GrpcInterface/GrpcInterface_Example/ExampleServer.cpp [hunk 2]
```diff
@@ -93,6 +93,30 @@ class ServerApp : public caffa::rpc::ServerApplication
...
+        {
+            if ( m_session )
+            {
+                if ( !m_session->isExpired() )
+                {
+                    return false;
+                }
+            }
+            return true;
+        }
```
Comment (kriben) [inline GrpcInterface/GrpcInterface_Example/ExampleServer.cpp:117]: Maybe collapse some of these if's?

**Review by kriben — APPROVED** [review]
Minor nitpicks.

File: GrpcInterface/cafGrpcObjectService.cpp (hunk 1 above)
Comment (lindkvis) [reply GrpcInterface/cafGrpcObjectService.cpp:133]: Fixing this for next versions without updating sub-modules now

**Pull request merged**